src/identity/components/userprofile/UserDefaults.tsx

```diff
@@ -46,16 +49,15 @@ export const UserDefaults: FC = () => {
 
   const {defaultAccountId, handleSetDefaultAccount, userAccounts} =
     useContext(UserAccountContext)
-  const quartzOrganizations = useSelector(selectQuartzOrgs)
+  const orgs = useSelector(selectQuartzActiveOrgs)
 
```

Comment:
Users should only be able to set 'active' orgs as their default org in the current account.